tests/unstable/db/sqla/public_query.py

```diff
@@ -341,7 +341,12 @@ def test_joinedload(self):
             if photos is not None:
                 self.assertEqual(set(p.photo for p in user.photos),
                                  set(photos))
-
+            users = query.limit(1).all()
```

Comment:
Does this triggers usage of `_compound_eager_statement()` method? I think it's worth to mention this is comments here.